pkg/proxy/iptables/proxier.go

```diff
@@ -1295,16 +1301,7 @@ func (proxier *Proxier) syncProxyRules() {
 					"-m", protocol, "-p", protocol,
 					"--dport", strconv.Itoa(svcInfo.NodePort()),
 				)
-				if !svcInfo.NodeLocalExternal() {
-					// Nodeports need SNAT, unless they're local.
-					proxier.natRules.Write(
-						"-A", string(policyClusterChain),
-						args,
-						"-j", string(KubeMarkMasqChain))
-				} else {
-					// TODO: Make all nodePorts jump to the firewall chain.
-					// Currently we only create it for loadbalancers (#33586).
```

Comment:
if you're killing this comment it might be good to explain why in the commit message for future git archeology reference?